pkgs/by-name/cr/craft-docs/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  undmg,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "craft-docs";
+  version = "2.8.8";
+
+  src = fetchurl {
+    name = "Craft.dmg";
+    url = "https://res.craft.do/native/sparkle/Craft.dmg";
+    hash = "sha256-5IlzPLzUZmMFEyL2tNDXjvb86/CW4dscVzbnNipKrek=";
```

Comment:
```suggestion
    hash = "sha256-M2NBmssSgGJxEx+T2nZ6pMiPFPbcJCuBK3r0frOJRHI=";
```
You really need an archived URL. Otherwise this package will break all the time. (I can "archive" it on my site if you don't like the Internet Archive.)

Reply:
It's not that I don't like any particular archive, I'm just not sure of the legality of it. See the issue linked a bit up. You are right about breakages, though.